### PR TITLE
Improve the doc of PACIS, DS and JointDist

### PR DESCRIPTION
**Pull request by adutfoy**
I took into account the review of @EmmanuelArdillon :

1. I improved the JointDistribution doc: instrumental marginal ands core: I closed the Issue #2817 . 
2. I improved the example *Create and draw multivariate distributions* : I plotted the instrumental marginals and the marginals in both cases where the core is and is not a copula. I drew the same graphs for the copula.
3. I improved the Directional sampling, RandomDirection, OrthogonalDirection, SamplingStrategy doc : I rewrote the rinciple of the algorithm, I introduced the proper notations, ...
4. I improved the doc of the PostAnalyticalControlledImportanceSampling.

### CODE REVIEW

File: python/src/DirectionalSampling_doc.i.in [hunk 1]
```diff
@@ -77,20 +73,25 @@ Examples
...
 >>> myFunction = ot.SymbolicFunction(['E', 'F', 'L', 'I'], ['-F*L^3/(3*E*I)'])
 >>> myDistribution = ot.Normal([50.0, 1.0, 10.0, 5.0], [1.0]*4, ot.IdentityMatrix(4))
->>> # We create a 'usual' RandomVector from the Distribution
->>> vect = ot.RandomVector(myDistribution)
->>> # We create a composite random vector
->>> output = ot.CompositeRandomVector(myFunction, vect)
->>> # We create an Event from this RandomVector
->>> myEvent = ot.ThresholdEvent(output, ot.Less(), -3.0)
->>> # We create a DirectionalSampling algorithm
+>>> input_RV = RandomVector(myDistribution)
```
Comment (jschueller) [inline python/src/DirectionalSampling_doc.i.in:80]: ot.RandomV...

File: python/src/DirectionalSampling_doc.i.in [hunk 2]
```diff
@@ -77,20 +73,25 @@ Examples
...
+>>> output_RV = ot.CompositeRandomVector(myFunction, input_RV)
+
+Create the event :math:`\cD_f = \{\inputRV \in \Rset^{\inputDim} \, | \,\model(\vect{x}) \leq -3.0\}`:
+>>> threshold = -3.0
+>>> myEvent = ot.ThresholdEvent(output_RV, ot.Less(), -3.0)
```
Comment (jschueller) [inline python/src/DirectionalSampling_doc.i.in:85]: could you rename these variables ? myEvent -> event

File: python/src/DirectionalSampling_doc.i.in (hunk 1 above)
Comment (adutfoy) [reply python/src/DirectionalSampling_doc.i.in:80]: yes... thanks!

File: python/src/DirectionalSampling_doc.i.in (hunk 2 above)
Comment (adutfoy) [reply python/src/DirectionalSampling_doc.i.in:85]: yes

File: python/src/PostAnalyticalControlledImportanceSampling_doc.i.in [hunk 3]
```diff
@@ -1,16 +1,116 @@
...
+Let :math:`\ell` be the function defined by: 
+
+.. math::
+
+    \ell(\vect{u}) = \vect{u}^*.\vect{u} - \beta^2
+
+where :math:`\vect{u}^*` is the design point in the standard space, and :math:`\beta` the distance
+of the design point from the origin of the standard space: :math:`\beta = \| \vect{u}^*\|`.
+
+The tangent hyperplan at the design point in the standard space is defined by the equation:
```
Comment (josephmure) [inline python/src/PostAnalyticalControlledImportanceSampling_doc.i.in:48]:
```suggestion
The tangent hyperplane at the design point in the standard space is defined by the equation:
```

File: python/src/SamplingStrategyImplementation_doc.i.in [hunk 4]
```diff
@@ -17,12 +19,17 @@ OT_SamplingStrategy_doc
 // ---------------------------------------------------------------------
 
 %define OT_SamplingStrategy_generate_doc
-"Generate the sample.
+"Generate the diretions.
```
Comment (josephmure) [inline python/src/SamplingStrategyImplementation_doc.i.in:22]:
```suggestion
"Generate the directions.
```

**Review by josephmure — CHANGES_REQUESTED** [review]

File: python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py [hunk 5]
```diff
@@ -13,40 +13,77 @@
...
-distribution = ot.JointDistribution(marginals)
-
-# %%
-# We can draw the PDF (here in dimension 2) :
+theta = 2.0
+cop = ot.ClaytonCopula(theta)
+distribution = ot.JointDistribution(marginals, cop)
+
+# %%
+# We can check here that the instrumental marginals are really the marginal distributions.
```
[the remark this answers is not in the document]
Comment (adutfoy) [reply python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py:44]: yes

File: python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py [hunk 6]
```diff
@@ -13,40 +13,77 @@
...
+# In the following graphs, we draw the instrumental marginals and the real marginals, obtained with
+# the method :meth:`~openturns.Distribution.getMarginal`.
+# First, we draw the probability density functions of each component. We recall that the
+# :meth:`~openturns.Distribution.drawPDF`
+# command just generates the graph data. It is the viewer module that enables the actual display.
```
[the remark this answers is not in the document]
Comment (adutfoy) [reply python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py:49]: yes... I acknowledge that I kept this sentence without really understand it..... I removed it!

File: python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py [hunk 7]
```diff
@@ -13,40 +13,77 @@
...
+graph_CDF_1.setLegends(['instrumental marg', 'marg'])
+graph_CDF_1.setTitle("Second component")
+view = otv.View(graph_CDF_1)
+
+# %%
+# At last, we check that the copula of the multivariate distribution is the specified core
+# which was a copula.
+cop_dist = distribution.getCopula()
+graph_cop = cop_dist.drawPDF()
+graph_cop.add(cop.drawPDF())
```
[the remark this answers is not in the document]
Comment (adutfoy) [reply python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py:81]: thanks! It is better!

File: python/src/PostAnalyticalControlledImportanceSampling_doc.i.in (hunk 3 above)
Comment (adutfoy) [reply python/src/PostAnalyticalControlledImportanceSampling_doc.i.in:48]: yes

File: python/src/SamplingStrategyImplementation_doc.i.in (hunk 4 above)
Comment (adutfoy) [reply python/src/SamplingStrategyImplementation_doc.i.in:22]: yes

**Comment by adutfoy** [comment]
@josephmure : Thank you for your remarks! I integrated them except for the first one....

File: python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py [hunk 8]
```diff
@@ -57,26 +94,83 @@
...
+graph_cop = cop_dist.drawPDF()
+cop_dist_draw = graph_cop.getDrawable(1)
+levels = cop_dist_draw.getLevels()
+graph_core = core_dir.drawPDF()
+core_draw = graph_core.getDrawable(0)
+core_draw.setLevels(levels)
+core_draw.setLineStyle('dashed')
+graph_cop.add(core_draw)
+graph_cop.setTitle('Distribution copula and core')
+view = otv.View(graph_cop)
```
[the remark this answers is not in the document]
Comment (adutfoy) [reply python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py]: yes

File: python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py [hunk 9]
```diff
@@ -13,40 +13,77 @@
...
-#   - a :class:`~openturns.Normal` distribution ;
-#   - a :class:`~openturns.Gumbel` distribution.
-
+# First case: the core is a copula
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# In this case, we use a core which is a copula. In that case, the instrumental marginals
+# are the marginals of the multivariate distribution.
+#
+# We first create the marginals of the distribution:
+#
```
[the remark this answers is not in the document]
Comment (adutfoy) [reply python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py]: ok

**Review by josephmure — APPROVED** [review]

**Pull request merged**